firedrake/solving.py

```diff
@@ -190,6 +198,9 @@ def _la_solve(A, x, b, **kwargs):
     r"""Solve a linear algebra problem.
 
     :arg A: the assembled bilinear form, a :class:`.Matrix`.
+    :arg P: an optional :class:`~.MatrixBase` to construct any
```

Comment:
This is in the wrong place in the docstring as it suggests that `P` is the second argument to the function. Should it be documented as a `kwarg`?

Reply:
Yes, I updated it.